ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -591,16 +608,23 @@ static bool ShouldHandleVnodeOpEvent(
         }
     }
     
-    *root = VirtualizationRoots_FindForVnode(vnode);
+    *rootIndex = VirtualizationRoot_FindForVnode(vnode);
+    
+    // This VNode is in the temp directory, do not act on it.
```

Comment:
Nit: I don't think this comment says anything that the following code doesn't already tell us. I would expand the comment to explain why (not what), or just drop it.